core/primitives/justification.hpp

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef KAGOME_JUSTIFICATION_HPP
+#define KAGOME_JUSTIFICATION_HPP
+
+namespace kagome::primitives {
+  /**
+   * Justification of the finalized block
+   */
+  struct Justification {};
```

Comment:
Why is it empty? Maybe you will leave a comment about it if you gonna keep it?

Reply:
Well, I have created this type to use, store, etc. My idea was that when somebody actually needs some of the fields of `Justification`, he/she will implement it

Reply:
For now make Justification as Buffer. This is how it in substrate and most likely it will be the same for us.
https://github.com/paritytech/substrate/blob/60c54f0d81857503aad1ad3166fe2f84a14205be/core/sr-primitives/src/lib.rs#L71

Reply:
I mean add 'Buffer data;' field